config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   resources :developers
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  post :search, to: 'developers#search'
+
```

Comment:
You should use `resources`.

Reply:
You can write as follows. (I didn't check the code works. But something like this)
```ruby
resources :developers do
  get :search, on: :collection
end
```

Reply:
Yep, I know, but in this case, I think don't need to add new route. Only use action `index` is fine
Because it used same a view. That is ok? @jp-ryuji

Reply:
@ntrung2006 
No problem.